packages/gafl-webapp-service/src/pages/contact/address/select/route.js

```diff
@@ -11,7 +12,7 @@ export const getData = async request => {
     isLicenceForYou,
     addresses,
     searchTerms,
-    lookupPage: ADDRESS_LOOKUP.uri,
```

Comment:
We need a test that the lookupPage result is set to the output of addLanguageCodeToUri - so mock it and use a Symbol